src/lib/__tests__/mergeSchemas.js

```diff
@@ -91,3 +92,53 @@ describe("convection link", () => {
     })
   })
 })
+
+describe("stiched schema regressions", () => {
+  it.only("union in interface fragment issue", async () => {
+    const artworkResponse = {
+      id: "banksy-di-faced-tenner-21",
+      sale_ids: ["foo-foo"],
+    }
+
+    const sales = [
```

Comment:
Nit: maybe call this `salesResponse` to be consistent with the one above it?

Reply:
Ok.